### PR TITLE
restrict bind eviction bind names to configured list

### PR DESCRIPTION
**Pull request by ModEtchFill**
We had some evictions which included general status instead of specific users.

### CODE REVIEW

File: lib/adaptivequemgr.go [hunk 1]
```diff
@@ -117,6 +118,19 @@ type BindCount struct {
 	Workers map[string]*WorkerClient // lookup by ticket
 }
 
+func bindEvictNameOk(bindName string) (bool) {
+	if len(GetConfig().BindEvictionNames) == 0 {
+		// for tests, allow all names to be subject to bind eviction
+		return true
+	}
+	for _, okSubname := range strings.Split(GetConfig().BindEvictionNames,",") {
+		if strings.Contains(bindName, okSubname) {
```
Comment (varuntechie) [inline lib/adaptivequemgr.go:127]: What if there are multiple matches ? for ex: account_number and account_number_two are part of the where clause?

Comment (varuntechie) [reply lib/adaptivequemgr.go:127]: should we also do case insensitive comparison?

Comment (varuntechie) [reply lib/adaptivequemgr.go:127]: Should we also add a warning message when the length of bind name is less than 7?

Comment (ModEtchFill) [reply lib/adaptivequemgr.go:130]: For multiple binds (account_number, account_number_two), both get evaluated for eviction. First one that matches goes through.

I've added case insensitive.

For short bind names, we'd probably log too much.

**Pull request merged**